test/e2e/BrsComponents.test.js

```diff
@@ -526,4 +526,25 @@ describe("end to end brightscript functions", () => {
             "5",
         ]);
     });
+
+    test.only("components/scripts/FieldChangeRunner.brs", async () => {
+        outputStreams.root = __dirname + "/resources";
+        await execute(
+            [resourceFile("components", "scripts", "FieldChangeRunner.brs")],
+            outputStreams
+        );
+
+        expect(allArgs(outputStreams.stdout.write).filter(arg => arg !== "\n")).toEqual([
+            "child: text field changed. new value:",
```

Comment:
Because of the way that we instantiate fields, these `onChange` callbacks are called on the initial set of the field. This is different than how RBI behaves (it doesn't call `onChange` for the initial value set), so I'm looking into how to refactor the call to make it not do this.

Reply:
This was happening because both the child and the parent had the field defined on them based on how we do component processing. Once I changed that, it worked!